spot.py

```diff
@@ -1,9 +1,19 @@
-# Super duper crude script to get quick results.  Sorry.
+"""
+Process LFMC data for a specific SPOT cube using onetile functions.
 
-# qsub -N SPOT-namadgi spot.qsub
-# or
-# qsub -v "FMC_SPOT_LOCATION=canberra" -l walltime=20:00:00 -N SPOT-canberra
-# spot.qsub
+The SPOT cube uses top of atmosphere radiance as opposed to corrected surface
+reflectance (which is used by MODIS).
```

Comment:
"This cube was assembled from data provided by Airbus using the notebook `SPOT-preprocessing.ipynb`" (check filename)